### ISSUE TITLE
模型训练无法得到README中的结果

### ISSUE THREAD

**Issue by zhangshaoyong** [liuheng92/tensorflow_PSENet#83]
十分感谢作者的代码，
现在我使用 ICDAR 2015 (training set) + ICDAR2017 MLT (training set)数据训练模型得不到README中的结果，但是使用作者训练的模型finetune能得到README中的结果。想请教一下你在训练的时候的参数是什么？是否有使用其他模型（例如rensnet18）进行参数初始化。谢谢

下面展示的loss的变化。基本已经不下降了。
I0610 20:33:12.249510 139815135364864 train.py:163] Step 041360, model loss 0.1846, total loss 0.2169, 0.89 seconds/step, 17.91 examples/second
I0610 20:33:21.638659 139815135364864 train.py:163] Step 041370, model loss 0.1146, total loss 0.1468, 0.94 seconds/step, 17.04 examples/second
I0610 20:33:30.717951 139815135364864 train.py:163] Step 041380, model loss 0.1721, total loss 0.2044, 0.91 seconds/step, 17.62 examples/second
I0610 20:33:39.667385 139815135364864 train.py:163] Step 041390, model loss 0.2126, total loss 0.2448, 0.89 seconds/step, 17.88 examples/second
I0610 20:33:48.386764 139815135364864 train.py:163] Step 041400, model loss 0.2107, total loss 0.2430, 0.87 seconds/step, 18.35 examples/second
I0610 20:33:58.035877 139815135364864 train.py:163] Step 041410, model loss 0.1223, total loss 0.1545, 0.96 seconds/step, 16.58 examples/second
I0610 20:34:07.956270 139815135364864 train.py:163] Step 041420, model loss 0.1818, total loss 0.2140, 0.99 seconds/step, 16.13 examples/second
I0610 20:34:16.916520 139815135364864 train.py:163] Step 041430, model loss 0.1226, total loss 0.1549, 0.90 seconds/step, 17.86 examples/second
I0610 20:34:26.354091 139815135364864 train.py:163] Step 041440, model loss 0.1766, total loss 0.2089, 0.94 seconds/step, 16.95 examples/second
I0610 20:34:34.790378 139815135364864 train.py:163] Step 041450, model loss 0.1508, total loss 0.1830, 0.84 seconds/step, 18.97 examples/second
I0610 20:34:44.537490 139815135364864 train.py:163] Step 041460, model loss 0.2026, total loss 0.2349, 0.97 seconds/step, 16.42 examples/second
I0610 20:34:54.426533 139815135364864 train.py:163] Step 041470, model loss 0.1505, total loss 0.1828, 0.99 seconds/step, 16.18 examples/second
I0610 20:35:03.461697 139815135364864 train.py:163] Step 041480, model loss 0.2120, total loss 0.2443, 0.90 seconds/step, 17.71 examples/second
I0610 20:35:11.972439 139815135364864 train.py:163] Step 041490, model loss 0.2796, total loss 0.3120, 0.85 seconds/step, 18.80 examples/second
I0610 20:35:21.587899 139815135364864 train.py:163] Step 041500, model loss 0.1493, total loss 0.1817, 0.96 seconds/step, 16.64 examples/second
I0610 20:35:32.514567 139815135364864 train.py:163] Step 041510, model loss 0.1236, total loss 0.1560, 1.09 seconds/step, 14.64 examples/second
I0610 20:35:43.676348 139815135364864 train.py:163] Step 041520, model loss 0.2090, total loss 0.2414, 1.12 seconds/step, 14.33 examples/second
I0610 20:35:52.411471 139815135364864 train.py:163] Step 041530, model loss 0.1214, total loss 0.1538, 0.87 seconds/step, 18.32 examples/second
I0610 20:36:01.599981 139815135364864 train.py:163] Step 041540, model loss 0.1417, total loss 0.1741, 0.92 seconds/step, 17.41 examples/second
I0610 20:36:11.521753 139815135364864 train.py:163] Step 041550, model loss 0.0881, total loss 0.1204, 0.99 seconds/step, 16.13 examples/second
I0610 20:36:20.682157 139815135364864 train.py:163] Step 041560, model loss 0.1370, total loss 0.1694, 0.92 seconds/step, 17.47 examples/second
I0610 20:36:29.963757 139815135364864 train.py:163] Step 041570, model loss 0.1688, total loss 0.2011, 0.93 seconds/step, 17.24 examples/second
I0610 20:36:39.568289 139815135364864 train.py:163] Step 041580, model loss 0.2142, total loss 0.2465, 0.96 seconds/step, 16.66 examples/second

**Comment by MichaelHL-ai** [comment]
https://github.com/liuheng92/tensorflow_PSENet/issues/58#issuecomment-566363925
